wifiphisher/common/victim.py

```diff
@@ -0,0 +1,189 @@
+"""
+This module was made to keep track the information of the victims connected to the rogue AP
+"""
+import os
+from macmatcher import MACMatcher as macmatcher
+
+dir_of_executable = os.path.dirname(__file__)
+path_to_project_root = os.path.abspath(os.path.join(dir_of_executable, '../../wifiphisher'))
+dir_of_data = path_to_project_root + '/data/'
+
+
+path_to_mac_file = dir_of_data + "wifiphisher-mac-prefixes"
+path_to_url_file = dir_of_data + "wifiphiser-os-initial-requests"
+
+
+#class to create a Victim when it connects to the AP
+class Victim(object):
+
+
+    def __init__(self, vmac_address):
+        self.vmac_address = vmac_address
+        self.ip_address = "Unknown"
+        self.hostname = "Unknown"
+        self.vendor = "Unknown"
+
+    def print_victim(self):
+        print (self.vmac_address, self.hostname, self.vendor)
+
+    """
+    receives a victims mac address as input, finds the corresponding vendor by using a macmacther object
+    and then accesses the victim dictionary and changes the vendor for the victim with the corresponding mac address
+    :param self: empty Victim instance
+    :type self: Victim
+    :param vmac_address: mac address of the victim
+    :type vmac_address: str
+    """
```

Comment:
Docstrings need to be under the definition of the method (under the `def` part).